contrib/html_demo/UICode/readme.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Cannot currently try out the site since we removed images from this PR to not have larger (non-text) files in github. This hence needs explanation how to download the resources (ie images) and where to put them. Also any other documentation, usage instructions, etc.